.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -24,22 +24,27 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots or videos to help explain your problem.
 
 **Desktop (please complete the following information):**
- - OS: [e.g. Windows 10, Wine]
+
+- OS: [e.g. Windows 10, Wine]
 
 ** FSMP**
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)